.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,280 @@
+name: Publish to PyPI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+
+jobs:
+  build-fable-library:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '9.0.x'
+      - name: Install Poetry and Maturin
+        run: |
+          pipx install poetry
+          pipx install maturin
+      - name: Transpile F# to Python
+        run: ./build.sh fable-library --python
+      - name: Install Dunamai
+        run: pipx install dunamai
+      - name: Set version
+        run: |
+          VERSION=$(dunamai from git --format "{base}{stage}{revision}" --pattern "^(?P<base>\d+\.\d+\.\d+)((-(?P<stage>alpha|beta|theta))\.(?P<revision>\d+))?$" --latest-tag)
```

Comment:
@MangelMaxime Here is the version generated from the latest tag

Reply:
Thanks